docs/getting-started-guides/centos/centos_manual_config.md

```diff
@@ -10,15 +10,15 @@ assignees:
 
 ## Prerequisites
 
-You need two machines with CentOS installed on them.
+You need a machine acting as master and as many CentOS 7 hosts as you would like, that act as cluster nodes.
```

Comment:
"To configure Kubernetes with CentOS, you'll need a machine to act as a master, and one or more CentOS 7 hosts to act as cluster nodes."